entrypoint.sh

```diff
@@ -15,53 +15,69 @@ SCRIPT_AFTER_MAKE_TEST="`pwd`/.github/ci-bionic/after_make_test.sh"
 
 cd $GITHUB_WORKSPACE
 
-echo ::group::Dependencies from binaries
+echo ::group::Install tools: apt
 apt update
 apt -y install wget lsb-release gnupg
 sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" > /etc/apt/sources.list.d/gazebo-stable.list'
 apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
-apt-get update
+apt-get update 2>&1
 apt -y install \
   cmake \
   build-essential \
   curl \
   g++-8 \
   git \
   cppcheck \
-  python3-pip \
-  $APT_DEPENDENCIES
+  python3-pip
+echo ::endgroup::
 
+echo ::group::Install tools: pip
 pip3 install -U pip vcstool colcon-common-extensions
 echo ::endgroup::
 
+echo ::group::Install tools: source
+git clone https://github.com/linux-test-project/lcov.git -b v1.14 2>&1
```

Comment:
I assume we need latest version for some reason. Could we add a comment here why are we installing this version instead of using the 1.13 version coming from Ubuntu?

Reply:
I believe it's related to gcov, see https://github.com/osrf/sdformat/pull/255#issuecomment-620712728. Maybe @scpeters can suggest the exact comment that could be added?

Reply:
I think we were just using `master` before, I'm not sure why the specific tag was introduced, but it happened here: https://github.com/osrf/sdformat/commit/9d7d19ce62f87b84b5f98738f10fcd2d25050e29

It looks like that was done on `osrf/sdformat`, but not on the `ignitionrobotics` repositories I checked.